### ISSUE TITLE
Source Map

### ISSUE THREAD

**Issue by fadi-george** [pd4d10/vite-plugin-svgr#4]
When I enable source-maps. I see the following in the terminal when I build:

`Sourcemap is likely to be incorrect: a plugin (vite:svgr) was used to transform files, but didn't generate a sourcemap for the transformation.`

**Comment by xSorc** [comment]
Maybe we should prevent error showing right now?
![image](https://user-images.githubusercontent.com/41228762/118121949-d9ab2500-b3fa-11eb-9497-b9a34e3bf7fe.png)

I've tested this fix - it works the same like now, but without alert

**Comment by gustaff-weldon** [comment]
@pd4d10 I'm on 0.3.0 and still seeing this error"
```
Sourcemap is likely to be incorrect: a plugin (vite:svgr) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
```

My `vite.config.ts`:
```
// https://vitejs.dev/config/
export default ({ mode }) => {
  return defineConfig({
    build: {
      brotliSize: false,
      sourcemap: true
    },
    plugins: [
      reactRefresh(),
      svgr()
    ],
    publicDir: false,
    resolve: {
      alias: {
        '@zen': resolve(__dirname, '/src/')
      },
      mainFields: ['module', 'jsnext']
    },
  });
};
```

**Comment by frankwallis** [comment]
I am seeing the same as @gustaff-weldon - this fix does not seem to have been included in the 0.30 release: https://unpkg.com/vite-plugin-svgr@0.3.0/lib/index.js
